### PR TITLE
.net framework 4.5 , for support Tls12

### PR DESCRIPTION
**Pull request by juster111**
support tls1.2

### CODE REVIEW

**Comment by Mead42** [comment]
Any word if this PR will get merged?

**Comment by juster111** [comment]
I have not got any word yet.

**Comment by vtml** [comment]
Agree, it will be great if this PR can be merged.

File: EdgeGridAuth/EdgeGridV1Signer.cs [hunk 1]
```diff
@@ -237,8 +237,9 @@ public Stream Execute(WebRequest request, ClientCredential credential, Stream up
         {
 
             //Make sure that this connection will behave nicely with multiple calls in a connection pool.
-            ServicePointManager.EnableDnsRoundRobin = true;
-            request = this.Sign(request, credential, uploadStream);
+            ServicePointManager.EnableDnsRoundRobin = true; 
+			ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12;
```
Comment (dshafik) [inline EdgeGridAuth/EdgeGridV1Signer.cs]: Indentation should be fixed here

File: EdgeGridAuth/EdgeGridV1Signer.cs [hunk 2]
```diff
@@ -237,8 +237,9 @@ public Stream Execute(WebRequest request, ClientCredential credential, Stream up
...
+			request = this.Sign(request, credential, uploadStream);
```
Comment (dshafik) [inline EdgeGridAuth/EdgeGridV1Signer.cs]: Indentations should be fixed here

File: EdgeGridAuth/EdgeGridAuth.csproj [hunk 3]
```diff
@@ -9,7 +9,7 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>Akamai.EdgeGrid.Auth</RootNamespace>
     <AssemblyName>Akamai.EdgeGrid.Auth</AssemblyName>
-    <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
```
Comment (dshafik) [inline EdgeGridAuth/EdgeGridAuth.csproj:12]: What impact does this have on users? Should this be considered a BC break?

Comment (juster111) [reply EdgeGridAuth/EdgeGridAuth.csproj:12]: https://stackoverflow.com/questions/28286086/default-securityprotocol-in-net-4-5

Comment (dshafik) [reply EdgeGridAuth/EdgeGridAuth.csproj:12]: @juster111 OK, so it's required for TLS 1.2, which means we can't avoid it, and users will be _required_ to update if necessary.

**Comment by dshafik** [comment]
Merged, thanks @juster111

**Pull request merged**